### PR TITLE
Update Simple SMR01 extract.R

### PR DESCRIPTION
**Pull request by CliveWG**
Does this need a dbDisconnect statement?

### CODE REVIEW

File: SMRA/Simple SMR01 extract.R [hunk 1]
```diff
@@ -38,3 +38,6 @@ smr01_query
 
 # Return the data
 smr01_extract <- collect(smr01_query)
+
+# close the connection
+dbDisconnect(smra_conn)
```
Comment (Moohan) [inline SMRA/Simple SMR01 extract.R:43]: If we've closed the connection, we may as well remove it too.
```suggestion
# close and remove the connection
dbDisconnect(smra_conn)
rm(smra_conn)
```

Comment (CliveWG) [reply SMRA/Simple SMR01 extract.R:43]: I like it James - makes perfect sense

**Review by Moohan — APPROVED** [review]
Thanks, Clive, I just committed my suggestion, so happy to approve this PR.

**Comment by CliveWG** [comment]
[like]  Clive Greatorex reacted to your message:
________________________________

**Pull request merged**